great_expectations/data_context/data_context.py

```diff
@@ -1680,29 +1683,67 @@ def get_validator(
             expectation_suite = self.create_expectation_suite(
                 expectation_suite_name=create_expectation_suite_with_name
             )
+        if (
+            sum(
+                bool(x)
+                for x in [batch_request is not None, batch_request_list is not None]
+            )
+            > 1
+        ):
+            raise ValueError(
+                "Only one of batch_request or batch_request_list may be specified"
+            )
+        if batch_request_list:
+            batch_list: List = []
+            for batch_request in batch_request_list:
+                batch_list.append(
+                    *self.get_batch_list(
```

Comment:
I could be wrong, but would it make sense to extend the `batch_list` here if `get_batch_list()` returns a list?
```suggestion
                batch_list.extend(
                    self.get_batch_list(
```

Reply:
Great point! I think these are functionally very similar, since I'm unpacking the returned list from `self.get_batch_list` with the `*`, but if it's clearer I can certainly update to use `extend`.